package.json

```diff
@@ -15,9 +15,9 @@
     "test:ci": "npm run test:sass && npm run test:javascript:ci",
     "test:sass": "mocha test/sass/test_sass.js",
     "test:javascript:transpile": "gulp sass:foundation && gulp test:transpile-js",
-    "test:javascript:phantomjs": "npm run test:javascript:transpile && mocha-phantomjs --ignore-resource-errors test/javascript/index.html",
+    "test:javascript:phantomjs": "npm run test:javascript:transpile && mocha-headless-chrome -a ignore-resource-errors -f test/javascript/index.html",
```

Comment:
We can rename that script since phantomjs is not used anymore. Maybe to `test:javascript:units` ?
Also, is there any downside or incompatibility with switching to `mochheadless-chrome` ?

Reply:
Right. I'll do this then or rewrite the commit.

Reply:
> Also, is there any downside or incompatibility with switching to mochheadless-chrome

Not that I knew of.